test/integration_tests/run_quick_test.py

```diff
@@ -58,6 +58,7 @@
         / "renate-integration-tests"
         / test_suite
         / job_name
+        / "0"
```

Comment:
What is this extra `0`?

Reply:
It's the seed: https://github.com/awslabs/Renate/pull/246/commits/50dab8fadeb949340e547a69bdfc50d8009ef2a3